UIDokan/Sources/UIDokan/Tools/CollectionviewHelper.swift

```diff
@@ -0,0 +1,27 @@
+//
```

Comment:
Let rename `CollectionViewHelper.swift` to `UICollectionView+DequeueHelper.swift` and move it to `Extensions`